Login/index.html

```diff
@@ -20,47 +20,53 @@
         </nav>
     </header> -->
     <div id="content">
-        <form action="Get">
+        <form action="Get"">
```

Comment:
extra quotation here.

Reply:
fixed

Reply:
Remove the action attribute altogether while you're at it @mnatour2.

Reply:
done